Makefile

```diff
@@ -130,6 +134,8 @@ OPENGL_LDFLAGS = -framework OpenGL -framework AGL
 endif
 endif
 
+OPENGLCOMPUTE_LDFLAGS = $(OPENGL_LDFLAGS)
```

Comment:
If the OpenGL compute ld flags are the same as OpenGL, maybe it should just piggyback off the WITH_OPENGL option and not have its own separate option?